src/bootloader/rollbacks/masked.h

```diff
@@ -37,7 +37,7 @@ class MaskedRollback : public Rollback {
       return;
     }
     std::string sink;
-    if (Utils::shell("fw_printenv bootfirmware_version", &sink) != 0) {
+    if (Utils::shell("fw_printenv -n bootfirmware_version", &sink) != 0) {
```

Comment:
iiuc, it's not an issue for this case https://github.com/foundriesio/aktualizr-lite/blob/18996f6d276661ee7971d21a6ce7a2d67e4295d0/src/bootloader/rollbacks/fiovb.h#L43 ?

Reply:
That is something that I will check today.

Reply:
I think that's not the case, fiovb doesn't print variable name, otherwise I would have stumbled upon the same issue when I was testing my closed imx8mmevk (but anyway worth checking)

@ricardosalveti